qa/tasks/cephfs/mount.py

```diff
@@ -111,6 +111,28 @@ def is_blacklisted(self):
                 return True
         return False
 
+    def create_file(self, filename='testfile', dirname=None, user=None,
+                    pwd=None, check_status=True):
```

Comment:
What purpose does "pwd" argument serve here?

Reply:
No idea. Looks like I was supposed to remove it.

Reply:
Are you going to? Let me know when your PR is done.

Reply:
I've removed it.

Can you please reply to this comment - https://github.com/ceph/ceph/pull/27481#issuecomment-482550708